AWS/deploy.sh

```diff
@@ -32,7 +31,7 @@ PS3="Please select desired collator to deploy: "
                         echo "terraform apply -var-file tfvars/${WORKSPACE}.tfvars"
                         aws s3 cp s3://${NAME_OF_THE_BUCKET}/terraform/tfvars/${WORKSPACE}.tfvars tfvars/${WORKSPACE}.tfvars --profile ${PROFILE}
                         terraform workspace select ${WORKSPACE}
-                        cat deployments/${WORKSPACE}.example > deployment.tf
+                        cat deployments/collator_with_command.example > deployment.tf
```

Comment:
@babebort-blaize move this version to /AWS/examples (with corresponding deployments and tfvars)
update deploy.sh (and other scripts) so they work with single collator